check_photo_backups.py

```
@@ -39,405 +42,637 @@
+                        fieldnames = parts
+                        continue
+                    else:
+                        # SAFETY: Reject TSV files without an explicit header row.
+                        # Column-count guessing risks misaligning fields, which could
+                        # cause size_bytes or content_hash to be read from the wrong
+                        # column — potentially marking files as safe to delete incorrectly.
+                        print(f"WARNING: Skipping {inv_path.name}: tab-separated data found but no 'filepath' header row. "
+                              f"This inventory may be corrupt or from an unsupported version.", file=sys.stderr)
+                        return
+
+                row = dict(zip(fieldnames, parts))
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)